landlab/components/erosion_deposition/erosion_deposition.py

```diff
@@ -130,6 +130,14 @@ class ErosionDeposition(_GeneralizedErosionDeposition):
             "doc": "Node array containing downstream-to-upstream ordered list of node IDs",
         },
         "sediment__flux": {
+            "dtype": float,
+            "intent": "out",
+            "optional": False,
+            "units": "m3/s",
+            "mapping": "node",
+            "doc": "Sediment flux (volume per unit time of sediment leaving each node)",
```

Comment:
The description of the *sediment__flux* field is now inconsistent with other *landlab* components. The *LateralEroder* says that *sediment__flux* is, *"Sediment flux (volume per unit time of sediment __entering__ each node)"* but *Space* says that it is, *"Sediment flux (volume per unit time of sediment __leaving__ each node)"*.

I'm not sure which description is correct but we need to either update the descriptions or change some field names to make things consistent.

Reply:
The new description is correct for ErosionDeposition and SPACE (both versions), but the old one is correct for LateralEroder! So we have a naming inconsistency... Here's a possible solution:
- change doc for sediment__flux to "Sediment flux (volume per unit time)"
- use sediment__influx if the flux is specified as coming into an element from adjacent elements
- use sediment__outflux if it is specified as leaving the element
- for backward compatibility, retain sediment__flux and have it refer to one of the above, whichever it did originally

Reply:
It seems, to me anyway, the best option would be to use two different names depending on whether sediment in flowing into or out of grid cells. The pain will be to maintain backward compatibility.

What do you think?

Reply:
Agreed. Names changed to sediment__influx and sediment__outflux. Pointer to one of these, as appropriate, with name sediment__flux is retained for backward compatibility until version 3.0.0+.